src/app/frontend/error/component.ts

```diff
@@ -27,11 +26,10 @@ import {NavService} from '../common/services/nav/service';
 })
 export class ErrorComponent implements OnInit {
   private error_: KdError;
-  constructor(private readonly nav_: NavService, private readonly route_: ActivatedRoute) {}
 
-  ngOnInit(): void {
-    this.nav_.setVisibility(false);
+  constructor(private readonly route_: ActivatedRoute) {}
```

Comment:
This way when entering error view, sidebar is still visible right?

Reply:
Yes, I can hide it if needed.

Reply:
It would be best to hide it in case we are entering this view. You could use injector to get this service.